src/components/ComponentSettingsModal.jsx

```diff
@@ -272,11 +281,12 @@ export default class ComponentSettingsModal extends Component {
                 {/* Category */}
                 <p className={elementStyles.formLabel}>Category Folder Name</p>
                 <div className="d-flex">
-                  <select className="w-100" id="category" value={category} onChange={this.changeHandler} disabled={!isNewFile}>
-                    {
+                  <select className="w-100" id="category" value={category} onChange={this.changeHandler} disabled={isCategoryDropdownDisabled(isNewFile, category)} >
```

Comment:
Using `category` to determine whether this field is disabled causes a bug when creating new files: once the user selects a category, the file is locked into that choice, so the user is unable to make a change afterwards, even though the file has not been created. We should use `prevCategory` from the state instead.

Reply:
fixed in [bf2adb0](https://github.com/isomerpages/isomercms-frontend/pull/206/commits/bf2adb0d48e4dcb163b47abdb186a7b720fc75d7)